lab-michael/test/test-tcp.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// const server = require('../server.js');
+const assert = require('assert');
+const net = require('net');
+const client = new net.Socket();
+
+// require('../model /client.js');
+
+describe('Connection to Server Module', function() {
```

Comment:
Your tests are scaffolded pretty well, but aren't actually testing anything yet, so still some work to do here.